### PR TITLE
Testing basic Conan requests functionality

### PR DESCRIPTION
**Pull request by ChGen**
Basic unit test for essential Conan slices. Checks the match with jsons provided.
JSONAssert is used for friendly json matching error messages and potential flexible json comparison.

### CODE REVIEW

**Comment by ChGen** [comment]
@olenagerasimova , please review.

File: src/test/java/com/artipie/conan/http/ConansEntityTest.java [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+        for (final String file : files) {
+            new TestResource(String.join("/", ConansEntityTest.DIR_PREFIX, file))
+                .saveTo(storage, new Key.From(file));
+        }
+        final Response response = factory.apply(storage).response(
+            new RequestLine(RqMethod.GET, request).toString(),
+            new Headers.From("Host", "localhost:9300"), Content.EMPTY
+        );
+        final String expected = Json.createReader(
+            Thread.currentThread().getContextClassLoader().getResourceAsStream(json)
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/conan/http/ConansEntityTest.java:151]: this expression can be simplified with `new TestResource(json).asStream()`

**Comment by ChGen** [comment]
@olenagerasimova , please review.

**Review by olenagerasimova — APPROVED** [review]
thanks

**Pull request merged**